python/pyspark/sql/readwriter.py

```diff
@@ -1196,39 +1097,13 @@ def json(self, path, mode=None, compression=None, dateFormat=None, timestampForm
         ----------
         path : str
             the path in any Hadoop supported file system
-        mode : str, optional
```

Comment:
mode is a general option

Reply:
yeah, so I documented this to the JSON data source options table, and removed here.